osu.Game/Screens/Edit/Compose/Components/SelectionBox.cs

```diff
@@ -235,13 +236,27 @@ private void addButton(IconUsage icon, string tooltip, Action action)
         private void addDragHandle(Anchor anchor) => AddInternal(new SelectionBoxDragHandle
         {
             Anchor = anchor,
-            X = getAdjustmentToCenterCircleOnBorder(anchor).X,
-            Y = getAdjustmentToCenterCircleOnBorder(anchor).Y,
+            X = dragCircleAdjustments[anchor].X,
+            Y = dragCircleAdjustments[anchor].Y,
             HandleDrag = e => OnScale?.Invoke(e.Delta, anchor),
             OperationStarted = operationStarted,
             OperationEnded = operationEnded
         });
 
+        /// <summary>
+        /// Adjust Drag circle to be centered on the center of the border instead of on the edge.
+        /// </summary>
+        private Dictionary<Anchor, Vector2> dragCircleAdjustments = new Dictionary<Anchor, Vector2>(){
```

Comment:
i honestly think the last version you had was better. would recommend a revert.

Reply:
To clarify: do you think the dictionary was better than the padding trick I used? Not 100% sure how to interpret this comment.

Reply:
Oh, somehow missed your commit. That is indeed much better.